_people/JayeNias.md

```diff
@@ -0,0 +1,12 @@
+---
+first_name: Jaye
+last_name: Nias
+title: Assistant Professor of Computer and Information Sciences
+school: Spelman College
+website: https://www.jayenias.com/
+image: https://static.wixstatic.com/media/4a7add_2ea535ea9ee74b3e8adfd1324bbec06e~mv2.jpg/v1/fill/w_468,h_603,al_c,q_80,usm_0.66_1.00_0.01/Pushing%2520through%2520the%2520challenges%2520_.webp
+---
+Prof. Nias is a researcher in computational social science in the Course-Based Undergraduate Research Experiences (CURE) Program. Her current research focus is on developing educational computational technologies to promote cultural diversity and societal inclusion. She has also conducted research in human computer interaction, with a focus on child-computer interactions and mobile computing.
+* [CultureBot: A Culturally Relevant Humanoid Robot Dialogue Agent](https://dl.acm.org/doi/10.1145/3374135.3385306)
```

Comment:
We still want the full citation (with authors, journal name, etc) just only have the title be a link. See other profiles for examples